Tests failing with latest version of requests?

Running tox on `master` has a few failing targets, the ones that pull down the current version of requests, which is 2.2.1 (released 2014-01-23).

```
  py26: commands succeeded
  py27: commands succeeded
  py33: commands succeeded
  pypy: commands succeeded
ERROR:   py26requests: commands failed
ERROR:   py27requests: commands failed
ERROR:   pypyrequests: commands failed
  py26oldrequests: commands succeeded
  py27oldrequests: commands succeeded
  pypyoldrequests: commands succeeded
```

Each of the failing targets fails with the following:

``` python
    def __init__(self, host, port=None, key_file=None, cert_file=None,
                 strict=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
                 source_address=None):
        HTTPConnection.__init__(self, host, port, strict, timeout,
>                               source_address)
E       TypeError: unbound method __init__() must be called with HTTPConnection instance as first argument (got HTTPSConnection instance instead)

/usr/local/Cellar/pypy/2.1.0/lib-python/2.7/httplib.py:1153: TypeError
```

I think the latest Travis CI builds on master will fail too once there is a commit, as I am seeing the same error in the Travis CI build for PR #58 from @asmundg.

--- a/vcr/patch.py
+++ b/vcr/patch.py
@@ -13,6 +13,7 @@ try:
     import requests.packages.urllib3.connectionpool as cpool
     _VerifiedHTTPSConnection = cpool.VerifiedHTTPSConnection
     _HTTPConnection = cpool.HTTPConnection
+    _HTTPSConnection = cpool.HTTPSConnection
 except ImportError:  # pragma: no cover
     pass
 

